Reading key-value pairs

```diff
@@ -0,0 +1,80 @@
+//Program that reads in a string of key-value pairs separated by semicolons and outputs the average value for a specific key.
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+int main() {
+    char str[1000];
+    char key[50];
+    int sum = 0, count = 0;
+    double avg;
+    printf("Enter key-value pairs separated by semicolon: ");
```

Comment:
give example of how the input should be

Reply:
i have mentioned that in the problem statement

Reply:
The problem statement is not known to the user